deployment/templates/cache-secret.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.development }}
```

Comment:
Is this file even used anymore? I thought we removed the cache in favor of the DB approach(?)